weave-js/src/components/UserLink.tsx

```diff
@@ -17,18 +18,15 @@ import {
 import {LoadingDots} from './LoadingDots';
 import {A, Link} from './PagePanelComponents/Home/Browse3/pages/common/Links';
 
-const FIND_USER_QUERY = gql`
-  query FindUser($username: String!) {
-    users(usernames: [$username], first: 1) {
-      edges {
-        node {
-          id
-          name
-          email
-          photoUrl
-          deletedAt
-        }
-      }
+const FIND_USERS_QUERY = gql`
```

Comment:
Why USERS plural?

Reply:
fixed in next commit. This was left over from my first attempt at doing a single query.